components/carousel/style/index.ts

```diff
@@ -47,7 +52,8 @@ const genCarouselStyle: GenerateStyle<CarouselToken> = (token) => {
         WebkitTapHighlightColor: 'transparent',
 
         '.slick-track, .slick-list': {
-          transform: 'translate3d(0, 0, 0)',
+          // Transforms should not be applied at all, if user has asked for it
+          ...(token.useTransform && { transform: 'translate3d(0, 0, 0)' }),
```

Comment:
`transform` props is auto generate with inline style by `react-slick` which is not owned by antd part.

Reply:
Yes, and when whe give `useTransform `as False to react slick it respects that and doesn't generate any transform

The problem is wit these styles in Antd it doesn't have any IF statement and just adds these to styles without checking if transforms are needed or not

https://react-slick.neostack.com/docs/api#useTransform